WTWSMS/common/traits/WtWSMS_traits.txt

```diff
@@ -141,11 +141,14 @@ legendary_leader = {
 
 the_prophet = {
 	
-	muslim_opinion = 25
-	martial = 6
+	muslim_opinion = 100
+	martial = 12
 	learning = 12
  
 	ai_ambition = 1000
+	ai_zeal = 1000
+	ai_honor = -500
+	ai_rationality = 1
```

Comment:
If there is an alternate way to give him a propensity to Intrigue, I'm all ears. Maybe adding the "Master Intriguer" Trait will get him to use it more?